xshop/users/api/views.py

```diff
@@ -9,6 +10,10 @@ class TokenApi(APIView):
 
     serializer_class = TokenApiSerializer
 
+    @swagger_auto_schema(
+        operation_description="Generate new token for user",
+        request_body=TokenApiSerializer,
+    )
```

Comment:
please create a TokenResponseSerializer to serializer the response data
![Uploading image.png…]()